fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/JobsServletSpec.groovy

```diff
@@ -155,6 +192,35 @@ class JobsServletSpec extends Specification {
         GroovyTestUtils.compareJson(result, expectedResponse, JsonSortStrategy.SORT_BOTH)
     }
 
+    def "jobs/pagination/results returns the number of results we requested through pagination parameters" () {
```

Comment:
Where is this segment coming from? Near as I can tell, we aren't storing a ticket called `pagination` anywhere in the `ApiJobStore`. Also, we shouldn't be using `pagination` as the ticket name. We should instead be using `ticket1` or `ticket2` or `ticket3` or something. For a second there, I thought we were doing something screwy, where we had to add an extra `pagination` path segment in order to paginate results!